src/data/DataManager.js

```diff
@@ -15,45 +15,49 @@ export default class DataManager extends Component {
             loading: true,
             data: {}
         }
-    } 
+    }
 
-    loadData(paths) {
+    loadData(paths, param = "") {
         let dataLoad = []
         for (var index in paths) {
-            dataLoad.push(dataSource.GetData(paths[index]))
+            paths[index] == 'user' ? dataLoad.push(dataSource.GetData(paths[index], param)) : dataLoad.push(dataSource.GetData(paths[index]))
```

Comment:
Maybe could be generalized to work for all endpoints with a param instead of just user. Maybe the load data function could accept a list of objects that hold the path and it's associated params.

Reply:
This is a good point, I definitely need to revisit this section of code.

Reply:
for this endpoint for account profile. is it possible to narrow the endpoints to 1 rather than 3?